### PR TITLE
UI/QA feed back

### PR DESCRIPTION
**Pull request by AbdullahAhmedSoomro**
Bug Fixes

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -28,6 +29,12 @@ void main() async {
     ),
   );
   await Injector.initialise();
+  SystemChrome.setSystemUIOverlayStyle(
+    const SystemUiOverlayStyle(
+      statusBarColor: Colors.transparent,
+      statusBarIconBrightness: Brightness.dark,
+    ),
+  );
```
Comment (hashirshoaeb) [inline lib/main.dart:37]: cannot we set it through theme?

File: lib/main.dart [hunk 2]
```diff
@@ -37,6 +44,7 @@ class MyApp extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return MaterialApp.router(
+      debugShowCheckedModeBanner: false,
```
Comment (hashirshoaeb) [inline lib/main.dart:47]: QA's demand?

**Review by hashirshoaeb — APPROVED** [review]

File: lib/main.dart (hunk 2 above)
Comment (AbdullahAhmedSoomro) [reply lib/main.dart:47]: Nope

**Review by hashirshoaeb — APPROVED** [review]

File: lib/main.dart (hunk 1 above)
Comment (AbdullahAhmedSoomro) [reply lib/main.dart:37]: Brightness.dark is basically an enum

Comment (hashirshoaeb) [reply lib/main.dart:37]: abi k liye theek hai per bad mai yehi tang kry ga. khair proceeding to merge it.

**Pull request merged**